tutorials/cap-service-deploy/cap-service-deploy.md

```diff
@@ -2,7 +2,7 @@
 author_name: René Jeglinsky
 author_profile: https://github.com/renejeglinsky
 title: Deploy a CAP Business Service to SAP Business Technology Platform
-description: This tutorial shows you how to deploy your SAP Cloud Application Programming Model (CAP) application to SAP Business Technology Platform, Cloud Foundry environment using SAP HANA Cloud service.
+description: This tutorial shows you how to deploy your SAP Cloud Application Programming Model (CAP) application to SAP BTP, Cloud Foundry environment using SAP HANA Cloud service.
```

Comment:
BTP should be spelled out the first time it is used (aside from the tutorial's title).

If you said "...application to SAP Business Technology Platform (BTP),...)" and later only used BTP it would be good.